model/common/src/icon4py/model/common/metrics/reference_atmosphere.py

```diff
@@ -210,3 +210,103 @@ def compute_d_exner_dz_ref_ic(
     Returns: first vertical derivative of reference exner pressure
     """
     return -grav / (cpd * theta_ref_ic)
+
+
+@field_operator
+def _compute_d2dexdz2_fac_mc(
+    theta_ref_mc: fa.CellKField[vpfloat],
+    inv_ddqz_z_full: fa.CellKField[vpfloat],
+    exner_ref_mc: fa.CellKField[vpfloat],
+    z_mc: fa.CellKField[wpfloat],
+    cpd: wpfloat,
+    grav: wpfloat,
+    del_t_bg: wpfloat,
+    h_scal_bg: wpfloat,
+) -> tuple[fa.CellKField[vpfloat], fa.CellKField[vpfloat]]:
+    """
+    Compute vertical derivative of d_exner_dz/theta_ref for full level mass points.
+
+    Args:
+        theta_ref_mc:
+        inv_ddqz_z_full:
+        exner_ref_mc:
+        z_mc:
+        cpd:
+        grav:
+        del_t_bg:
+        h_scal_bg:
+
```

Comment:
do we need this docstring if all fields are anyway already described in the program?